index_head.html

```diff
@@ -4,20 +4,20 @@
 <head>
 
 	<!-- THE BIG META -->
-	<title>E adesso cosa succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive</title>
+	<title>E ora che succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive</title>
 	<meta name="description" content="una guida interattiva" /> 
 	<link rel="icon" type="image/png" href="favicon.png">
 	<meta charset="utf-8">
 	<base target="_blank">
 
 	<!-- Twitter Card data -->
 	<meta name="twitter:card" content="summary_large_image">
-	<meta name="twitter:title" content="E adesso cosa succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
+	<meta name="twitter:title" content="E ora che succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
 	<meta name="twitter:description" content="una guida interattiva">
 	<meta name="twitter:image" content="https://ncase.me/covid-19/sharing/thumbnail.png">
 
 	<!-- Open Graph data -->
-	<meta property="og:title" content="E adesso cosa succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
+	<meta property="og:title" content="E ora che succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
 	<meta property="og:type" content="website">
 	<meta property="og:url" content="https://ncase.me/covid-19/">
 	<meta property="og:image" content="https://ncase.me/covid-19/sharing/thumbnail.png">
```

Comment:
I believe this should be: <meta property="og:image" content="https://harisont.github.io/covid-19/sharing/thumbnail.png">

Reply:
also the og:url is referencing the English version

Reply:
Absolutely true, would you like to submit those changes?